lib/cocoapods/validator.rb

```diff
@@ -1108,7 +1108,7 @@ def xcodebuild(action, scheme, configuration, deployment_target:)
         command += Fourflusher::SimControl.new.destination(:oldest, 'tvOS', deployment_target)
       when :visionos
         command += %w(CODE_SIGN_IDENTITY=- -sdk xrsimulator)
-        command += Fourflusher::SimControl.new.destination(:oldest, 'visionOS', deployment_target)
+        command += Fourflusher::SimControl.new.destination(:oldest, 'xrOS', deployment_target)
```

Comment:
Apple has been changing all xrOS references to visionOS. Does this still work with Xcode 15.1? Should Fourflusher be changed instead?

Reply:
Is this fix sufficient? https://github.com/CocoaPods/Core/pull/763

Reply:
I tested this in Xcode 15.1 Beta 3 with `xrOS` and it worked 🤷‍♂️ But I'm happy to make the change in `Fourflusher` instead!